integrationtest/java/modules/orm-lucene/src/main/java/module-info.java

```diff
@@ -0,0 +1,33 @@
+@SuppressWarnings( "requires-transitive-automatic" )
```

Comment:
Looks like this warning is right, and we shouldn't be using `requires transitive` on an automatic module, because such a module never has any dependencies?

Reply:
this is so bizarre. I could've sworn that it was falling with all kinds of "missing Lucene classes" errors until I put that transitive in place even, as you are saying, it shouldn't really make a difference. I've cleaned it all now, and it works just fine 🤷🏻‍♂️ 🪄